src/modules/auth/auth.controller.ts

```
@@ -55,4 +56,18 @@ export class AuthController {
   logout(@Req() req: Request) {
     return this.authService.logout(req);
   }
+
+  @Post('/update-password')
```

ESLint

[error] Replace `··` with `↹`

(prettier/prettier)